tests/inputs/res/setting.xsd

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema version="1.0"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
+
+  <!-- THIS SETTING XSD IS WRONG BUT MUST KEEP ITS NAME FOR TESTS PURPOSES -->
```

Comment:
Would be nice to not duplicate this file

Reply:
This file is not duplicated.